pkg/build/api/types.go

```diff
@@ -59,6 +59,10 @@ type BuildSpec struct {
 	// Compute resource requirements to execute the build
 	Resources kapi.ResourceRequirements
 
+	// PostCommit is a build hook executed after the build output image is
+	// committed, before it is pushed to a registry.
+	PostCommit BuildPostCommitSpec
```

Comment:
Why is this not just `post` like deployments?  Will we ever add `postPush`?  What other possible `posts` are there.